sql/core/src/main/java/org/apache/spark/sql/sources/v2/DataSourceOptions.java

```diff
@@ -27,6 +27,39 @@
 /**
  * An immutable string-to-string map in which keys are case-insensitive. This is used to represent
  * data source options.
+ *
+ * Each data source implementation can define its own options and teach its users how to set them.
+ * Spark doesn't have any restrictions about what options a data source should or should not have.
+ * Instead Spark defines some standard options that data sources can optionally adopt. It's possible
+ * that some options are very common and many data sources use them. However different data
+ * sources may define the common options(key and meaning) differently, which is quite confusing to
+ * end users.
+ *
+ * The standard options defined by Spark:
+ * <table>
+ *   <tr>
+ *     <th><b>Option key</b></th>
+ *     <th><b>Option value</b></th>
+ *   </tr>
+ *   <tr>
+ *     <td>path</td>
+ *     <td>A comma separated paths string of the data files/directories, like
+ *     <code>path1,/absolute/file2,path3/*</code>. Each path can either be relative or absolute,
+ *     points to either file or directory, and can contain wildcards. This option is commonly used
+ *     by file-based data sources.</td>
+ *   </tr>
+ *   <tr>
+ *     <td>table</td>
+ *     <td>A table name string representing the table name directly without any interpretation.
```

Comment:
what do you mean by "without any interpretation"?

Reply:
It means it's a pure string, there is not parsing rule for it like SQL identifier. I put some examples below and hopefully they can explain it well.

Reply:
I think this is clear with the examples.